src/Raven.Server/NotificationCenter/OutOfMemoryNotifications.cs

```diff
@@ -13,52 +13,15 @@ public class OutOfMemoryNotifications
     {
         private readonly TimeSpan _updateFrequency = TimeSpan.FromSeconds(15);
         private readonly ConcurrentDictionary<string, NotificationTime> _notificationTimes = new ConcurrentDictionary<string, NotificationTime>();
-        private readonly ConcurrentDictionary<(StorageEnvironment, Type), NotificationTime> _notificationsMetadata = new ConcurrentDictionary<(StorageEnvironment, Type), NotificationTime>();
+        private readonly ConcurrentDictionary<(string, Type), NotificationTime> _notificationsMetadata = new ConcurrentDictionary<(string, Type), NotificationTime>();
```

Comment:
I don't like this much.
The problem is that this is never cleaned, and may end up being a problem area all on its own.